packages/shared/src/hooks/useCoresFeature.ts

```diff
@@ -36,5 +37,15 @@ export const useCanAwardUser = (
 };
 
 export const useCanPurchaseCores = (): boolean => {
-  return useHasAccessToCores() && !isIOSNative();
+  const hasAccess = useHasAccessToCores();
+
+  if (!hasAccess) {
+    return false;
+  }
+
+  if (isIOSNative()) {
+    return iOSSupportsCoresPurchase();
```

Comment:
This means we are not going to allow purchases through web with apple (in-app browser to be specific). Although it is more of an edge case. Should we maybe not check anymore since we will support both Paddle and `iap` with iPhone products?

Reply:
I am fine with both, probably just to check with product.

Reply:
Cool. I'll send a message. Just to be sure we won't get into any legal trouble.